sql/mysql_db/privileged_database_provider.go

```diff
@@ -164,58 +164,12 @@ func (pdb PrivilegedDatabase) Name() string {
 
 // GetTableInsensitive implements the interface sql.Database.
 func (pdb PrivilegedDatabase) GetTableInsensitive(ctx *sql.Context, tblName string) (sql.Table, bool, error) {
```

Comment:
Can we get rid of this type in a followup PR?

Reply:
Looking into it, we rely on this to get the `mysql` database, so I need to figure out a good way to retain access to it